src/test/java/com/puppycrawl/tools/checkstyle/PackageObjectFactoryTest.java

```diff
@@ -91,16 +99,37 @@ public void testMakeCheckFromName()
     }
 
     @Test
-    public void testCreateObjectWithIgnoringProblems() throws Exception {
-        final Method createObjectWithIgnoringProblemsMethod =
-                PackageObjectFactory.class.getDeclaredMethod(
-                        "createObjectWithIgnoringProblems", String.class, Set.class);
-        createObjectWithIgnoringProblemsMethod.setAccessible(true);
-        final Set<String> secondAttempt = new HashSet<>();
-        secondAttempt.add("com.puppycrawl.tools.checkstyle.checks.naming.ConstantNameCheck");
-        final ConstantNameCheck check = (ConstantNameCheck) createObjectWithIgnoringProblemsMethod
-                .invoke(factory, "ConstantName", secondAttempt);
-        assertNotNull(check);
+    public void testCreateModuleWithNonExistName() throws CheckstyleException {
+        testCreateModuleWithNonExistName("NonExistClassOne");
+        testCreateModuleWithNonExistName("NonExistClassTwo");
```

Comment:
It is for codecov. Around `createModule`, the lazy init, `if (thirdPartyNameToFullModuleName == null)` condition will be only touched as one status (null) if we only test once. Maybe the coverage would be solved when re-manage the order of finding modules.